c++/src/Reader.hh

```diff
@@ -187,6 +188,16 @@ namespace orc {
     friend class TestRowReader_advanceToNextRowGroup_Test;
     friend class TestRowReader_computeBatchSize_Test;
 
+    inline uint64_t lastRowOfCurrentStripe() const {
```

Comment:
Please add a function description comment.

Reply:
Done

Reply:
The function has remove.